File: Tech Module SoftUni Java/ArraysExercise/src/dayOfWeek.java
import java.util.Scanner;

public class dayOfWeek {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String[] days = {"Monday",
                "Tuesday",
                "Wednesday",
                "Thursday",
                "Friday",
                "Saturday",
                "Sunday"};

        int day = Integer.parseInt(scanner.nextLine());

        if (day > 0 && day < 8) {
            System.out.println(days[day-1]);
        } else {
            System.out.println("Invalid day!");
        }

//        Enter a day number [1…7]
//        and print the day name (in English) or "Invalid day!". Use an array of strings.
    }
}

File: JS Core SoftUni/Javascript Advanced January 2020/JSSyntaxAndFunctions/VSCode SyntaxAndFunctions/dayOfWeek.js
function solve(day) {

    let days = {
        'Monday': 1,
        "Tuesday": 2,
        "Wednesday": 3,
        "Thursday": 4,
        "Friday": 5,
        "Saturday": 6,
        "Sunday": 7
    };
    if (day !== "Monday" && day !== 'Tuesday' && day !== 'Wednesday' && day !== 'Thursday' && day !== 'Friday' && day !== 'Saturday' && day !== 'Sunday') {
        console.log(`error`);
    } else {
        console.log(days[day]);

    }
}

solve("Wednesday");